src/test/java/ancaMarian/tests/tema5/DemoQA.java

```diff
@@ -0,0 +1,222 @@
+package ancaMarian.tests.tema5;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class DemoQA {
+    private static String url;
+    private static ChromeDriver driver = null;
+
+
+    DemoQA(String url, ChromeDriver driver) {
+        this.url = url;
+        this.driver = driver;
+        driver.get(url);
+    }
+
+
+    public static void checkTextBox() {
+        WebElement textBox = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[1]/span"));
```

Comment:
When using xpath as a selector try to avoid using absolute xpath(meaning that you start from the html tag) and use relative xpath(meaning that you'll try to minimize the xpath by starting with the closest unique parent and continuing from there) instead. For example "/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[1]/span" would become "//ul[@class='menu-list']/li[1]/span[text()='Text Box']"

Reply:
class is not with capital letter there. it seems that there is a user with this username

Reply:
done, I have replaced the absolute xpath with "//*[@id='item-0']/span"